logger.go

```diff
@@ -144,7 +144,7 @@ func (log *logger) Check(lvl Level, msg string) *CheckedEntry {
 	ce.ErrorOutput = log.errorOutput
 
 	if log.addCaller {
-		ce.Entry.Caller = MakeEntryCaller(runtime.Caller(log.callerSkip))
+		ce.Entry.Caller = MakeEntryCaller(runtime.Caller(log.callerSkip + 2))
```

Comment:
Can we preserve the `defaultCallerSkip` constant instead of putting a literal `2` here?